doc/docs/master/how-tos/index.md

```diff
@@ -75,10 +75,14 @@ data operations in Pachyderm.
       </div>
       <div class="mdl-card__actions mdl-card--border">
         <ul>
+           <li><a href="https://docs.pachyderm.com/latest/concepts/pipeline-concepts/datum/glob-pattern/#test-your-datums" class="md-typeset md-link">
```

Comment:
Small change, I would probably just say `Test Datums` since we also don't say `Create your pipeline`